<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Author_Detail_Widget</class>
 <widget class="QWidget" name="Author_Detail_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>F2:根据作者名查询</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/DS_2020_GUI/image/astronomy-clouds-dark-evening.jpg)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="name">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>701</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url();
color:rgb(255, 255, 255);
font: 20pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>701</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="382,1000">
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QListView" name="lvw_collaborator">
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image:url();
background-color:rgba(244,244,244,100)</string>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListView" name="lvw_info">
       <property name="cursor" stdset="0">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image:url();
background-color:rgba(244,244,244,100)</string>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>560</y>
      <width>701</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url();
background-color:rgba(244,244,244,100)</string>
    </property>
    <property name="text">
     <string>！ 点击左边可以查看与该合作者相关的文章， 点击右边Info可以跳转到对应的文章</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="DS_2020_GUI.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>lvw_info</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>Author_Detail_Widget</receiver>
   <slot>showInfoDetail(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>610</x>
     <y>345</y>
    </hint>
    <hint type="destinationlabel">
     <x>873</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showInfoDetail(QModelIndex)</slot>
 </slots>
</ui>
